lib/plugins/aws/lib/naming.js

```diff
@@ -245,6 +245,12 @@ module.exports = {
   getUsagePlanKeyLogicalId(usagePlanKeyNumber) {
     return `ApiGatewayUsagePlanKey${usagePlanKeyNumber}`;
   },
+  getStageLogicalId(stage) {
+    const normalizedStage = this.normalizeName(stage)
+      .replace(/-/g, 'Dash')
+      .replace(/_/g, 'Underscore');
+    return `ApiGatewayStage${normalizedStage}`;
```

Comment:
Do we need to include the stage name (e.g. `dev`) here?

Should this be setup that one API can have multiple stages? Otherwise we might want to switch to `ApiGatewayStage` as the logical id...

/cc @eahefnawy @dschep

Reply:
Hmm. Yeah i don't think it's necessary

Reply:
Sounds reasonable! Thanks @dschep 👍

Any obligations @eahefnawy?

Reply:
Just removed this...